### DISCUSSION TITLE
I'm Chinese. I want to know that I can't find the satellite names of my own country, such as Tianzhou and Tianhe. Please help us solve it?

### DISCUSSION THREAD

**Discussion by 1234f503** [Stellarium/stellarium#2518] [Q&A] [answered]
I'm Chinese. I want to know that I can't find the satellite names of my own country, such as Tianzhou and Tianhe. Please help us solve it?

**Comment by alex-w** [comment, marked as answer]
![stellarium-032](https://user-images.githubusercontent.com/88731/179443890-88e0633d-470b-411f-af5b-5242432cb408.jpeg)

Probably both was not added in your list of satellites. Please check TLE sources for space stations and add wanted satellites into list of satellites (+ check the visibility parameter)

**Reply by 1234f503** [reply]
thank

**Reply by 1234f503** [reply]
I wish you a happy Chinese New Year!

**Reply by 1234f503** [reply]
This is a blessing from China. Happy New Year!
